packages/legacy/core/App/screens/ProofChangeCredential.tsx

```diff
@@ -0,0 +1,176 @@
+  })
+
+  useEffect(() => {
+    setLoading(true)
+
+    credProofPromise
+      ?.then((value) => {
+        if (value) {
+          const { groupedProof, retrievedCredentials } = value
+          setLoading(false)
+          const activeCreds = groupedProof.filter((proof) => altCredentials.includes(proof.credId))
+          const credList = activeCreds.map((cred) => cred.credId)
+          const formatCredentials = (
+            retrievedItems: Record<string, (AnonCredsRequestedAttributeMatch | AnonCredsRequestedPredicateMatch)[]>
+          ) => {
+            return Object.keys(retrievedItems)
+              .map((key) => {
+                return {
+                  [key]: retrievedItems[key].filter((attr) => credList.includes(attr.credentialId)),
+                }
+              })
+              .reduce((prev, curr) => {
+                return {
+                  ...prev,
+                  ...curr,
+                }
+              }, {})
+          }
+          const selectRetrievedCredentials: AnonCredsCredentialsForProofRequest | undefined = retrievedCredentials
+            ? {
+                ...retrievedCredentials,
+                attributes: formatCredentials(retrievedCredentials.attributes) as Record<
+                  string,
+                  AnonCredsRequestedAttributeMatch[]
+                >,
+                predicates: formatCredentials(retrievedCredentials.predicates) as Record<
+                  string,
+                  AnonCredsRequestedPredicateMatch[]
+                >,
+              }
+            : undefined
+          setRetrievedCredentials(selectRetrievedCredentials)
+          setProofItems(activeCreds)
+        }
+      })
+      .catch((err: unknown) => {
+        const error = new BifoldError(
+          t('Error.Title1026'),
+          t('Error.Message1026'),
+          (err as Error)?.message ?? err,
+          1026
+        )
+        DeviceEventEmitter.emit(EventTypes.ERROR_ADDED, error)
+      })
+  }, [])
+
+  const listHeader = () => {
+    return (
+      <View style={{ ...styles.pageMargin, marginVertical: 20 }}>
+        {loading ? (
+          <View style={styles.cardLoading}>
+            <RecordLoading />
+          </View>
+        ) : (
+          <Text style={TextTheme.normal}>{t('ProofRequest.MultipleCredentials')}</Text>
+        )}
+      </View>
+    )
+  }
+
+  const changeCred = (credId: string) => {
+    onCredChange(credId)
+    navigation.goBack()
+  }
+  const hasSatisfiedPredicates = (fields: Fields, credId?: string) =>
+    proofItems.flatMap((item) => evaluatePredicates(fields, credId)(item)).every((p) => p.satisfied)
+
+  return (
+    <SafeAreaView style={styles.pageContainer} edges={['bottom', 'left', 'right']}>
+      <FlatList
+        data={proofItems}
+        ListHeaderComponent={listHeader}
+        renderItem={({ item }) => {
+          return (
+            <View style={styles.pageMargin}>
+              <TouchableOpacity
```

Comment:
I think these are deprecated now. Can you switch it to Pressable?

Reply:
done

Reply:
Now that you're using `Pressable` does it still have the ripple / active opacity effect when pressed? Curious